python/ray/serve/http_proxy.py

```diff
@@ -204,6 +205,18 @@ def get_handle(name):
             description="The number of HTTP requests processed.",
             tag_keys=("route", ))
 
+        self.request_error_counter = metrics.Counter(
+            "serve_num_http_error_requests",
+            description="The number of non-200 HTTP responses.",
+            tag_keys=("route", ))
+
+        self.deployment_request_error_counter = metrics.Counter(
+            "serve_num_deployment_http_error_requests",
+            description=(
+                "The number of non-200 HTTP responses returned by each "
+                "deployment."),
+            tag_keys=("deployment", ))
```

Comment:
maybe also tag with error code?

Reply:
`serve_num_http_error_requests ` tag with error code will be better, the deployment only return 500 in this version of code.